deploy/helm/flightctl/templates/flightctl-api-networkpolicy.yaml

```
@@ -1,4 +1,4 @@
-{{ if not .Values.api.nodePorts.api }}
+{{ if eq (include "flightctl.getServiceExposeMethod" .) "route" }}
```

```suggestion
{{ if neq (include "flightctl.getServiceExposeMethod" .) "nodePort" }}
```
for when we hopefully add support for ingress&friends